cmd/schema/main.go

```diff
@@ -49,6 +49,7 @@ func main() {
 	registry.Register(&flowsv1.Sequence{})
 	registry.Register(&flowsv1.Parallel{})
 	registry.Register(&eventingv1alpha1.EventPolicy{})
+	registry.Register(&eventingv1alpha1.EventTransform{})
```

Comment:
was this used to generate the actual CRD?

Asking because other new CRDs (e.g. `JobSink` or `Integration` _source/sink_ are missing.
(I can add them in a separate PR)

Reply:
but it's only for bootstrapping it to give an head start

`go run ./cmd/schema/ dump EventTransform`